src/Configuration/SnowflakeTableNodesDecorator.php

```diff
@@ -37,8 +37,6 @@ protected function addItemsNode(NodeBuilder $nodeBuilder): void
                     ->scalarNode('size')->beforeNormalization()->always(fn($v) => (string) $v)->end()->end()
                     ->scalarNode('nullable')->end()
                     ->scalarNode('default')->end()
-                    ->scalarNode('foreignKeyTable')->cannotBeEmpty()->end()
-                    ->scalarNode('foreignKeyColumn')->cannotBeEmpty()->end()
```

Comment:
Tím by se dalo celý týhle třídy zbavit a klidně to validování přesunout do db-writer-config balíčku

Reply:
Takže přenést do `TableNodesDecorator::addItemsNode()` tu validaci, že nemůžou být všechny fields ignored?

https://github.com/keboola/db-writer-snowflake-gcs/blob/09d3cf7fa4fb81038dd997ced7bb64985c62d60d/src/Configuration/SnowflakeTableNodesDecorator.php#L17

Reply:
jo jasně, to by mohlo (mělo) platit pro všechny writery, ne jen pro snflk